pytket/pytket/backends/backendinfo.py

```diff
@@ -13,13 +13,127 @@
 # limitations under the License.
 
 """ BackendInfo class: additional information on Backends """
+
+from ast import literal_eval
 from dataclasses import dataclass, field, asdict
 from typing import Any, Dict, List, Optional, Set, cast, Tuple, Union
 
 from pytket.routing import Architecture, FullyConnected  # type: ignore
 from pytket.circuit import Node, OpType  # type: ignore
 
 
+def _serialize_all_node_gate_errors(
+    d: Optional[Dict[Node, Dict[OpType, float]]]
+) -> Optional[Dict[str, Dict[int, float]]]:
+    if d is None:
+        return None
+    return {
+        str(n.to_list()): {ot.value: err for ot, err in errs.items()}
+        for n, errs in d.items()
+    }
+
+
+def _deserialize_all_node_gate_errors(
+    d: Optional[Dict[str, Dict[int, float]]]
+) -> Optional[Dict[Node, Dict[OpType, float]]]:
+    if d is None:
+        return None
+    return {
+        Node.from_list(literal_eval(n)): {
+            OpType(int(ot)): err for ot, err in errs.items()
```

Comment:
This doesn't match how Ciruit serializes optypes, where it uses the string of the optype (partially to try and keep the serialized form stable against the enum numbering changing).

Reply:
Right. This'll probably have to be exposed via the binders as I don't think it is currently.